packages/forms-web-app/__tests__/unit/controllers/projects/documents/filters/dates/config/expected-missing-date-values-dates-filter-obj.js

```diff
@@ -0,0 +1,68 @@
+const expectedMissingDateValuesDatesFilterObj = {
+	activeDateFilters: [],
+	datesFilter: [
+		{
+			errorSummary: {
+				list: [
+					{
+						href: '#docments-page-date-from-form-group',
+						text: 'The published from date must include year'
+					},
+					{
+						href: '#docments-page-date-to-form-group',
+						text: 'The published to date must include day and month'
+					}
+				],
+				title: 'There is a problem'
+			},
+			formGroups: [
+				{
+					errorMessage: { text: 'The published from date must include year' },
+					errorMessageText: 'The published from date must include year',
+					errorMessageTitle: 'published from',
+					id: 'docments-page-date-from-form-group',
+					inputNamePrefix: 'date-from',
+					inputs: [
+						{ classes: 'govuk-input--width-2', name: 'day', value: '2' },
+						{ classes: 'govuk-input--width-2', name: 'month', value: '1' },
+						{ classes: 'govuk-input--width-4 govuk-input--error', name: 'year', value: '' }
+					],
+					name: 'date-from',
+					title: 'From'
+				},
+				{
+					errorMessage: { text: 'The published to date must include day and month' },
+					errorMessageText: 'The published to date must include day and month',
+					errorMessageTitle: 'published to',
+					id: 'docments-page-date-to-form-group',
+					inputNamePrefix: 'date-to',
+					inputs: [
+						{ classes: 'govuk-input--width-2 govuk-input--error', name: 'day', value: '' },
+						{ classes: 'govuk-input--width-2 govuk-input--error', name: 'month', value: '' },
+						{ classes: 'govuk-input--width-4', name: 'year', value: '2023' }
+					],
+					name: 'date-to',
+					title: 'To'
+				}
+			],
+			isOpen: true,
+			title: 'Date published',
+			type: 'date'
+		}
+	],
+	datesFilterErrorSummary: {
+		list: [
+			{
+				href: '#docments-page-date-from-form-group',
+				text: 'The published from date must include year'
+			},
+			{
+				href: '#docments-page-date-to-form-group',
+				text: 'The published to date must include day and month'
```

Comment:
This error message should be - The date must include day and month

Where has "published" come from